### PR TITLE
Improved Folder Structure and Additional Instance Type Functionalities 

### PR DESCRIPTION
**Pull request by jonrwong-amazon**
*Issue #, if available:*

*Description of changes:*
Added additional instance types for Bert and Mistral models. New instances include CPU (m8g, r8g, c8g) for Mistral and (c7g and m8g) for Bert. There is also improved folder structure, making it easy to choose a model of choice, decide which instance type to use, and follow the README files within that instance/model folder to continue deploying onto the EKS cluster. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: app/run-sd.sh
Comment (yahavb) [inline app/run-sd.sh:1]: shouldn't matplotlib be installed in both cases and not just xla, cuda or triton? we can move it up or even port it to the dockerfile as it does not need accelerated compute instance

File: bert/bert-c7g/README.md
Comment (yahavb) [inline bert/bert-c7g/README.md:1]: better if we have simpler directory structure like `bert` and under it single readme that calls out the instance type.

**Review by yahavb — COMMENTED** [review]
lets have a simpler directory structure pls. no need for sub directory just for instance type.

File: app/run-sd.sh
Comment (jonrwong-amazon) [reply app/run-sd.sh:1]: Yes, moved to dockerfile and uncommented out matplotlib import lines in respective .py files in /app.

File: bert/bert-c7g/README.md
Comment (jonrwong-amazon) [reply bert/bert-c7g/README.md:1]: Flattened directory and added single README for each model folder

**Review by yahavb — APPROVED** [review]

**Pull request merged**